### ISSUE TITLE
Repo link is broken

### ISSUE THREAD

**Issue by niekarends** [telefonicaid/fiware-cygnus#1520]
The repo url returns a 404:
http://repositories.lab.fiware.org/repo/rpm/6/

**Comment by fgalan** [comment]
Which particular file (I guess that some piece of documentation, in the form of .md file) has that broken link, please? Thx!

**Comment by niekarends** [comment]
I've found it in multiple places:

- https://github.com/telefonicaid/fiware-cygnus/blob/master/doc/cygnus-common/installation_and_administration_guide/install_with_rpm.md

-https://github.com/telefonicaid/fiware-cygnus/blob/master/doc/cygnus-ngsi/installation_and_administration_guide/install_with_rpm.md

There might be more. I am not sure what it should be.

**Comment by fgalan** [comment]
Yes, it seems that URL (which belong to FIWARE Foundation repositories) is no longer valid. Let's hope they can fix it.

CC: @caa06d9c

**Comment by fgalan** [comment]
@niekarends PR https://github.com/telefonicaid/fiware-cygnus/pull/1525 should have fixed documentation. Please have a new look and close the issue if everything is ok now or provide additional feedback if not. Thanks!
